easybuild/easyconfigs/g/Ghostscript/Ghostscript-9.16-goolf-1.7.20.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'ConfigureMake'
+
+name = 'Ghostscript'
+version = '9.16'
+
+homepage = 'http://ghostscript.com'
+description = """Ghostscript is a versatile processor for PostScript data with the ability to render PostScript to
+ different targets. It used to be part of the cups printing stack, but is no longer used for that."""
+
+toolchain = {'name': 'goolf', 'version': '1.7.20'}
+toolchainopts = {'pic': True}
+
+source_urls = ["http://downloads.ghostscript.com/public/"]
+sources = [SOURCELOWER_TAR_GZ]
+
+checksums = ['829319325bbdb83f5c81379a8f86f38f']
+
+dependencies = [
+    ('zlib', '1.2.8'),
+    ('libpng', '1.6.17'),  
+    ('freetype', '2.6'),
+    ('fontconfig', '2.11.94'),
+    ('GLib', '2.45.2'),
+    ('libjpeg-turbo', '1.3.1'),  # note: libjpeg-turbo 1.4.[01] doesn't work
```

Comment:
That's OK, all required deps are there, this PR just includes an extra easyconfig for libjpeg-turbo, which is fine.